lib/one_model.ex

```diff
@@ -141,38 +141,40 @@ defmodule OneModel do
       @doc """
       Get a list of #{@schema}'s.
 
-      ## Options'
+      ## Options
 
       * `preload: list`
+      * `order_by: atom | list`
       """
       @spec list(Keword.t()) :: [Struct.t()]
       def list(opts \\ []) do
         if preload = opts[:preload] do
           @schema
           |> preload(^preload)
-          |> order_by(asc: :inserted_at)
-          |> @repo.all
+          |> do_order(opts[:order_by])
+          |> @repo.all()
         else
-          @repo.all(@schema)
+          @schema
+          |> do_order(opts[:order_by])
+          |> @repo.all()
         end
       end
 
       @doc """
-      Get a list of #{@schema},s given a list of field value pairs.
+      Get a list of #{@schema},s given a list of {field, value} pairs.
 
       ## Preload
 
       Pass a list of preloads with the `:preload` key.
 
       ## Examples
 
-          #{@schema}.list_by field1: value1, field2: field2, preload: [:association]
+          #{@schema}.list_by(field1: value1, field2: value2, preload: [:association])
       """
-      @spec list_by(Keyword.t()) :: List.t()
+      @spec list_by(Keword.t()) :: [Struct.t()]
```

Comment:
I'm not sure why your changes are showing this change. It's almost as if you based this from master and then put back the old changes. This line should already be like this in the pre-dialyzer version I think. Weird. But let's not revert this to the bad version, regardless of the way it got here. Restore the line with the spelling fix and the List.t.